lib/SILOptimizer/Transforms/SILMem2Reg.cpp

```diff
@@ -872,15 +1015,8 @@ void StackAllocationPromoter::addBlockArguments(BlockSetVector &phiBlocks) {
   LLVM_DEBUG(llvm::dbgs() << "*** Adding new block arguments.\n");
 
   for (auto *block : phiBlocks) {
-    // The stored value.
+    // The stored value or its lexical move.
```

Comment:
Don't we need to pass both as phi arguments anymore ?

Reply:
No, the moved-from value is consumed by the move.